apps/educational_institutions/models.py

```diff
@@ -103,8 +109,9 @@ def __repr__(self):
                           Column('shortname', VARCHAR(20)),
                           Column('main_email', VARCHAR(50)),
                           Column('university_id', INTEGER),
-                          Column('dekan_id', INTEGER),
-                          Column('dekan_full_name', VARCHAR(255)))
+                          Column('dean_id', INTEGER),
+                          Column('dean_full_name', JSON))
```

Comment:
Why did you add this field as JSON type?

Reply:
see changes im main.sql file